templates/example_qc_metric_eval.ipynb

```diff
@@ -3,6 +3,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    fovs = [fov_match[0] for f in fovs if len(fov_match := re.findall(r'fov-\d+-scan-\d+', f)) == 1]
</pre><p>Why do these need to be sorted? I thought the aggregation function handles that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>This isn't a sorting function, it was to prevent extra <code>.json</code> files (ex. <code>2022-01-22_adam_control_tissue_2675V.json</code> in <code>2022-01-22_adam_control_tissue_2675V</code>) from making it in. That being said, I can just use the <code>.bin</code> files to extract the names. </p>

Reply:
Oh I see now, I'll also need to adjust the comment.